vendor/bat-native-ads/src/bat/ads/internal/bundle.cc

```diff
@@ -188,6 +193,50 @@ std::unique_ptr<BundleState> Bundle::GenerateFromCatalog(
   return state;
 }
 
+bool Bundle::DoesOsSupportCreativeSet(
+    const CreativeSetInfo& creative_set) {
+  if (creative_set.oses.empty()) {
```

Comment:
I _really_ don't like this `oses` - is this an existing convention? But, I can't think of anything better other than `operating_systems`...

Reply:
@masparrow This is existing code and comes from the catalog feed